### ISSUE TITLE
Can't delete history

### ISSUE THREAD

**Issue by lucyfernandez** [frigus02/RESTer#19]
The trash button on history is not working. I **can't remove/delete the history**!
I do some test which involve oAuth authentication and now my secret ID is shown explicitly in the history and I can remove it.
I already remove, restart firefox and then reinstall this addon but the history is still there.
Please fix it.
FYI, I'm using Firefox 49.0.1

**Comment by frigus02** [comment]
Hi. Thanks for reporting, I can reproduce this bug and will probably upload a fix on the weekend.

To be honest, I did not test that addon data is removed, when the addon is removed. I always assumed it would be done automatically. But because it isn't I will also take care about this.

If you want an immediate solution, you can find the addon data in your Firefox profile directory. Mine is here:
```
C:\Users\XXX\AppData\Roaming\Mozilla\Firefox\Profiles\XXXXXX.default\storage\permanent\indexeddb+++rester-at-kuehle-dot-me
```

When you close Firefox, delete the folder `indexeddb+++rester-at-kuehle-dot-me` and start Firefox again, RESTer should be clean. However this method does delete everything including saved requests, not just the history.

**Comment by frigus02** [comment]
Ok. I fixed the first part, which is deleting a single history entry. I will push out an update tomorrow with a few other fixes.

However, deleting all data when the addon is uninstalled is not possible. This is because there is no way for an addon to know that is has been (or will be) uninstalled. For more information see the yellow box on the bottom of [this page](https://developer.mozilla.org/en-US/Add-ons/SDK/Tutorials/Listening_for_load_and_unload#reason). This might work though, as soon as I migrated the addon to WebExtensions (#18).
